src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -594,7 +596,7 @@ partial class BoundIncrementOperator : IIncrementExpression
 
     partial class BoundBadExpression
     {
-        protected override OperationKind ExpressionKind => OperationKind.None;
+        protected override OperationKind ExpressionKind => OperationKind.BadExpression;
```

Comment:
I agree that 'Invalid'  would be fine and that would be what I would have chosen. (We must have turned off any policy against it long ago in the framework)

FWIW, we have also have at least one precedent for 'Bad' in .NET: BadImageFormatException. :)